app/src/main/plugins.js

```diff
@@ -50,13 +50,7 @@ class Plugins {
     pkg.dependencies[name] = 'latest';
     fs.writeFileSync(pkgPath, JSON.stringify(pkg));
 
-    try {
-      await this.upgrade();
-    } catch (err) {
-      console.error(err);
-      console.error(err.stderr);
-      console.error(err.stdout);
-    }
+    await this.upgrade();
```

Comment:
Will this never throw: Don't we want the silent logging here? 🤔

Reply:
@albinekb It can throw if `npm install` fails. Because it's in an async function it will be propagated and reported here: https://github.com/wulkano/kap/pull/290/files/5b6a4309b5e5ef2f3e85b345bf004c13a1454e37#diff-6f1bf0a59b91e2427edb5a476a341460R625

Reply:
I see, thanks! :+1: